src/frameworks/web/components/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components';
+import theme from 'theme';
+import React from 'react';
+// eslint-disable-next-line no-unused-vars
+import { IButton } from 'interfaces/frameworks/web/components/atoms/Button/IButton';
+
+interface IBaseButton {
+  isPrimary: boolean
+  onClick: any
+}
+
+const BaseButton = styled.button`
+  font-family: 'Muli', 'Noto Sans KR', sans-serif;
+  background-color: ${(props: IBaseButton) => (props.isPrimary ? theme.color.primary.Scarlet : theme.color.primary.Black)};
+  color: ${theme.color.primary.White} !important;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  border-radius: 10px;
+  font-size: 16px;
+  transition: all 0.2s;
+  
+  &:hover {
+     background-color: ${(props: IBaseButton) => (props.isPrimary ? theme.color.primary.Salmon : theme.color.secondary.Nickel)};
+  }
+`;
+
+const DefaultButton = styled(BaseButton)`
+    width: 154px;
+    height: 56px;
+    color: ${(props) => props.color ?? theme.color.primary.Black};
+`;
+
+const FieldButton = styled(BaseButton)`
+    width: 154px;
+    height: 48px;
+    color: ${(props) => props.color ?? theme.color.primary.Black};
+`;
+
+const SmallButton = styled(BaseButton)`
+    width: 116px;
+    height: 40px;
+    font-size: 14px;
+    color: ${(props) => props.color ?? theme.color.primary.Black};
+`;
+
+const WideButton = styled(BaseButton)`
+    width: 350px;
+    height: 70px;
+    color: ${(props) => props.color ?? theme.color.primary.Black};
+`;
+
+export default class Button extends React.PureComponent<IButton> {
+  render() {
+    const { type } = this.props;
+    const { label } = this.props;
+    const { isPrimary } = this.props;
+    const { onClick } = this.props;
```

Comment:
이거 한 줄로 펴서 받지 않은 이유가 있나요?

```suggestion
    const { type, label, isPrimary, onClick } = this.props;
```

Reply:
수정 완료!